### ISSUE TITLE
Unable to install

### ISSUE THREAD

**Issue by romanomsandra** [daniel-j/sigil-plugin-ibooksfix#1]
Hi there!
I cannot install this plugin in Sigil 0.9.17. Could you please give more detailed instructions about how to achieve it? Many thanks.

**Comment by rghartwig** [comment]
There is a problem with the Makefile.  The output ZIP file must have an underscore after the plugin name rather than a hyphen.  The output ZIP file should be named: iBooksFix_0.1.zip

Then from Sigil, select Mange Plugins from the Plugins menu, then click Add Plugin and select the plugin ZIP file.

**Comment by romanomsandra** [comment]
Sigil stil says that this is not a valid plugin... Could you please produce a directly installable zip of the plugin? Many, many thanks!

**Comment by rghartwig** [comment]
I've attached my version, hopefully.

To make it, you can unzip the zip file from the distribution and then zip just the iBooksFix folder from there, then rename it to "iBooksFix_0.1.zip" and that one should be installable.

I tried this plugin and I think it worked, but didn't fix the index problem for me.  I haven't tested it enough to be sure, though.  Such an irritating problem in Apple's iOS reader!  It works on the desktop.

[iBooksFix_0.1.zip](https://github.com/daniel-j/sigil-plugin-ibooksfix/files/3561492/iBooksFix_0.1.zip)

**Comment by romanomsandra** [comment]
Many thanks for your help. Now I can install it, but it crashes when I try to execute it in Sigil. The error message says:

Status: failed

Traceback (most recent call last):
  File "/Applications/Sigil.app/Contents/plugin_launchers/python/launcher.py", line 134, in launch
    target_script = __import__(script_module)
  File "/Users/Sandra/Library/Application Support/sigil-ebook/sigil/plugins/iBooksFix/plugin.py", line 13, in <module>
    from lxml import etree
ImportError: No module named lxml
Error: No module named lxml

Any idea? Thanks a lot.

**Comment by rghartwig** [comment]
That's an error from Python, and I don't know much about that, but you might try installing the latest version of Python on your Mac.  I did this some time ago without problems.  That will probably fix it (it runs OK for me on my iMac).  Version 2.7 comes on the Mac and 3.7 is the latest.  You can get it here:  https://www.python.org/downloads/mac-osx/

**Comment by romanomsandra** [comment]
Great, thanks! I'll try tomorrow.

**Comment by romanomsandra** [comment]
That was correct. I changed the Python to auto in Sigil, and it worked perfectly. 
However, the fix didn't resolve the problem. Still, when syncing an epub to an iPhone from iCloud, the TOC was wrong. 
I finally resolved it sharing the original epub to both devices at the same time with AirDrop from my computer. Now both have the same file, with correct TOC, and sync rightly. It is an strange bug, really.
Many thanks for your help!
